packages/rag/generate_questions_pipeline.py

```
@@ -0,0 +1,262 @@
+    cost: CostTracker,
+) -> tuple[list[dict], list[dict], int]:
+    """단일 카테고리에 대해 문제 생성 및 저장
+
+    Returns:
+        (합격 문제 리스트, 탈락 문제 리스트, 저장된 문제 수)
+    """
+    # 1. 청크 검색 + Reranker
+    try:
+        retrieval = retrieve_chunks_with_reranker(category, top_k=10)
+        if not retrieval.chunks or retrieval.question_count == 0:
+            logger.log(f"관련 청크 없음 (스킵)", indent=1)
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)